src/Menu.java

```diff
@@ -0,0 +1,34 @@
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.nio.file.Files.exists;
+import static java.nio.file.Files.readAllLines;
+import static java.nio.file.Path.of;
+
+public class Menu extends Constants{
+
+    public void Run(){
+        System.out.print(MENU_INFO);
+        String in = scanner.nextLine();
+        List<String> list ;
+
+            //Вибір напрямку програми
+            if (in.equals("1")) {
```

Comment:
"1","2" and other numbers should be like constants:
private(or public) static final String ENCODING_ITEM = "1";